django/contrib/postgres/search.py

```diff
@@ -33,6 +35,24 @@ def db_type(self, connection):
         return 'tsquery'
 
 
+class SearchConfig(Expression):
+    def __init__(self, config):
+        super().__init__(output_field=None)
+        if not (config and hasattr(config, 'resolve_expression')):
```

Comment:
The following is likely more readable given `config` works on `None` as well.

```suggestion
        if not hasattr(config, 'resolve_expression'):
```